src/chunk.js

```diff
@@ -83,7 +83,23 @@ export function chunk(nodeOrAmpDoc, fn, priority) {
  * @return {!Chunks}
  */
 export function chunkInstanceForTesting(nodeOrAmpDoc) {
-  return fromClassForDoc(nodeOrAmpDoc, 'chunk', Chunks);
+  return getChunkServiceForDoc_(nodeOrAmpDoc);
+}
+
+/**
+ * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
+ * @return {!Chunks}
+ */
+function getChunkServiceForDoc_(nodeOrAmpDoc) {
+  installChunkServiceForDoc_(nodeOrAmpDoc);
+  return getServiceForDoc(nodeOrAmpDoc, 'chunk');
+}
+
+/**
+ * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
+ */
+export function installChunkServiceForDoc_(nodeOrAmpDoc) {
```

Comment:
Does this need to be exported? Note that trailing underscore is a naming convention for private props/functions.

If it's called in only one place, you probably don't need the method to start with.